File: Python/ExampleTests/Test_Calculator.py
import pytest
from common import *
from calculator_glue import CalculatorGlue

# --- Calculation Tests ---

def test_Calculation_AddTwoNumbers():
    glue = CalculatorGlue()
    object_list_1 = [
        AdderString('2', '3', '5'),
        AdderString('10', '20', '30'),
        AdderString('-1', '1', '0'),
    ]
    glue.examples_Calculation_AddTwoNumbers(object_list_1)

File: Rust/ExampleTests/test_calculator.rs
#![allow(unused_mut, unused_variables, unused_imports)]

use crate::common::*;
use super::calculator_glue::CalculatorGlue;

// --- Calculation Tests ---

#[test]
fn calculation_add_two_numbers() {
    let mut glue = CalculatorGlue::new();
    glue.examples_calculation_add_two_numbers(&[
        AdderString::from_vec(&["2", "3", "5"]),
        AdderString::from_vec(&["10", "20", "30"]),
        AdderString::from_vec(&["-1", "1", "0"]),
    ]);
}

